assets/style.css

```diff
@@ -0,0 +1,53 @@
+body {background: linear-gradient(to bottom right,#242024,#0f0b0f); font-family: verdana;;}
```

Comment:
There's something a bit odd going on with the background. This is from the Waterway Arcane Fest Data page:

![image](https://user-images.githubusercontent.com/19622123/90526312-17597680-e125-11ea-9304-a62b9cda39b0.png)

I'm happy to approve the PR regardless, but since you took the trouble of giving it a classy gradient background, figured I'd let you know!